theme/themes/eea/modules/accordion.overrides

```diff
@@ -141,4 +141,11 @@
 .ui.accordion.tertiary .title:not(.active):hover,
 .ui.accordion.tertiary .title:not(.active):hover i {
   color: @tertiaryColorCSSVar;
-}
\ No newline at end of file
+}
+
+/* override slate-metadata addon styles */
+.ui.accordion .content .ui.list.metadata > .item .content,
+.description {
```

Comment:
@nileshgulia1 don't leave .description rule as there are several elements that have .description class
and they would all be affected, if you meant to modify the description of accordion then add the
necessary parent classes